src/ralph/lib/transitions/forms.py

```diff
@@ -13,6 +13,9 @@
 )
 
 
+TRANSITION_TEMPLATES = getattr(settings, 'TRANSITION_TEMPLATES', None)
```

Comment:
why getattr? add it to base settings and use `TRANSITION_TEMPLATES = settings.TRANSITION_TEMPLATES` here